<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriverPanel</class>
 <widget class="QMainWindow" name="DriverPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>576</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #F7F9FE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>500</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-radius: 20;
background-color: #256489;
color: #ffffff</string>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>829</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delivery Order</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="productSearchBar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>839</width>
      <height>49</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>72</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background: #E5E8ED;
border-bottom: 1px solid #71787E;
padding: 0 10;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="placeholderText">
     <string>Search Stock...</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>881</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background: #E5E8ED;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>40</width>
      <height>49</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #E5E8ED;
background-image: url(:/Default/Icons/Res/search.svg);
background-repeat: no-repeat;
background-position: center;
border-bottom: 1px solid #71787E;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="stockScroll">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>881</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="stockScrollContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>881</width>
       <height>221</height>
      </rect>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>881</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="productLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
